addons/account/models/account_reconcile_model.py

```diff
@@ -732,15 +732,15 @@ def _get_invoice_matching_amls_candidates(self, st_line, partner):
         :param st_line: A statement line.
         :param partner: The partner associated to the statement line.
         """
+        def create_order(alias=None):
```

Comment:
Name is a bit misleading, at first I thought this was about a sale order. I'd rename this helper function to `get_order_by_clause` or something like that